langchain/src/chat_models/universal.ts

```diff
@@ -238,7 +238,10 @@ interface ConfigurableModelFields extends BaseChatModelParams {
   queuedMethodOperations?: Record<string, any>;
 }
 
-class _ConfigurableModel<
+/**
+ * Internal class used to create chat models.
```

Comment:
Not sure if this is set up for langchain yet, but it's good practice to use the `@internal` tsdoc tag on internal structures, that way we don't need to treat them as public for the purpose of breaking changes.

Reply:
```suggestion
 * Internal class used to create chat models.
 *
 * @internal
```